src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -193,6 +192,10 @@ public static Integer getHearingDurationBenefitIssueCodes(SscsCaseData caseData,
         }
     }
 
+    public static boolean isPoAttending(SscsCaseData caseData) {
```

Comment:
Where has this come from?

Reply:
Or, rather, where is it used?

It's nice in the sense that its naming convention makes it clear how to determine if a "po" is "attending". Still, I'd like to know what's the plan around it before accepting the PR.

Reply:
It was added to be used for another ticket, removed the code now. Again, result of an incorrect rebase (apologies)